simp_le.py

```diff
@@ -78,6 +82,26 @@ class TestCase(unittest.TestCase):
     """simp_le unit test case."""
 
 
+@contextlib.contextmanager
+def temp_umask(umask):
+    """Context manager that temporarily sets the process umask."""
+    oldmask = os.umask(umask)
+    try:
+        yield
+    finally:
+        os.umask(oldmask)
+
+
+def open_keyfile(filename, mode):
```

Comment:
that's not specific to keys, and is used generically elsewhere when `sensitive=True`, so please rename to `open_sensitive`